### DISCUSSION TITLE
Optimize token

### DISCUSSION THREAD

**Discussion by casanos0v0** [lcmd65/assistant-dsc.web#3] [Q&A] [answered]
How that index instruction encoder and match

**Comment by lcmd65** [comment, marked as answer]
it a simple method: first I match this user text  with each index instruction by fuzzy and word2vec (combine), it seem not encoder, but syntax is import dance in this method

**Reply by casanos0v0** [reply]
Ok thanks

**Comment by casanos0v0** [comment]
> it a simple method: first I match this user text with each index instruction by fuzzy and word2vec (combine), it seem not encoder, but syntax is import dance in this method

**Comment by casanos0v0** [comment]
Ok
